README.md

```diff
@@ -18,6 +18,13 @@ For details on the JSON Schema used by UBI to send and receive queries and event
 * [Query Response Schema](https://o19s.github.io/ubi/docs/html/query.response.schema.html)
 * [Event Schema](https://o19s.github.io/ubi/docs/html/event.schema.html)
 
+## UBI, Data Prepper, and Open Telemetry
+
+The UBI plugin can store UBI query data in one of two ways:
+
```

Comment:
```suggestion
- By sending the UBI query data as a document index operation directly to a OpenSearch `ubi_queries` index in the same cluster as the plugin.
```

Reply:
Do we want to add that method back?

Reply:
I know it's not a "super scalable" approach, but wasn't it the easiest way to trial it?

Reply:
It was. I think we removed it prior to the first release due to requirements around where to store the UBI data. With the Data Prepper endpoint, you can store it back in the same OpenSearch or any other OpenSearch. Should we reconsider adding it as a third option?